applications/asset_tracker/prj.conf

```diff
@@ -40,6 +40,7 @@ CONFIG_CLOUD_API=y
 CONFIG_NRF_CLOUD=y
 CONFIG_NRF_CLOUD_LOG_LEVEL_DBG=y
 CONFIG_NRF_CLOUD_AGPS=y
+#CONFIG_NRF_CLOUD_CELL_POS=y
```

Comment:
I'd just remove this line since `NRF_CLOUD_CELL_POS` is disabled by default.

Reply:
I'll remove this line then ask for merge in Teams group